### DISCUSSION TITLE
warm start support

### DISCUSSION THREAD

**Discussion by jd-lara** [Sienna-Platform/PowerSimulations.jl#792] [Q&A]
@odow this is the other place where there is a call to MOI.supports to check if warm_start is allowed by the solver. I remember we added this because if we tried to warm start and the solver didn't support it there was an exception. 

https://github.com/NREL-SIIP/PowerSimulations.jl/blob/e73fb500f0fd5a1e6edbe2aa6b234da153263479/src/core/optimization_container.jl#L53

**Comment by odow** [comment]
Yes, this is correct.
